ProiectPractica/Controller/ConfigForm.cs

```diff
@@ -0,0 +1,74 @@
+using ProiectPractica.Model;
+
+namespace ProiectPractica
+{
+    public partial class ConfigForm : Form
+    {
+        public ConfigForm()
+        {
+            InitializeComponent();
+        }
+
+        /// <summary>
+        /// Still in progress
```

Comment:
aici m-am gandit sa scriu ca este inca in lucru, ca lumea sa stie ca nu e gata si sa nu o foloseasca.
In cazul in care scriam descrierea "normala" a functiei, cineva putea sa o foloseasca doar pe baza descrierii
si atunci ar fi cod care nu functioneaza ok.

Reply:
Inteleg ce ai vrut sa obtii. In mod ideal, ar trebui sa nu avem PR-uri in care codul "nu e gata". Daca am scris o metoda, aceasta ar trebui sa fie folositoare pentru o anumita functionalitate. In mod ideal, atata timp cat metoda isi face treaba pentru functionalitatea curenta, nu ar trebui sa specifici ca nu ar fi gata. 

Practic, uneori se mai intampla sa stim ca urmeaza si alta functionalitate, si atunci putem lasa un //TODO specific in zona care credem ca va fi schimbata. Dar in general ar trebui ca codul livrat sa fie de sine statator, sa nu fie situatii de genul.

Reply:
As recomanda sa pui un comentariu ok si daca vrei neaparat, sa mentionezi intr-un comentariu unde ar trebui ajustat pe viitor.